lsp-completion.el

```diff
@@ -495,19 +495,18 @@ Others: TRIGGER-CHARS"
 
     (cond
      ((equal lsp-completion-provider :none))
-     ((and (null lsp-completion-provider)
-           (functionp 'company-lsp))
-      (progn
-        (company-mode 1)
-        (add-to-list 'company-backends 'company-lsp)
-        (setq-local company-backends (remove 'company-capf company-backends))))
+
+     ((and lsp-completion-provider
+           (fboundp 'company-mode)
+           (member 'company-lsp company-backends))
+      (message "Warning: company-lsp is not supported anymore, make sure `lsp-completion-provider` is configured correctly."))
```

Comment:
can you use push company-capf and change the wording that we are going to use company-capf even if there was company-capf